config/sql/config.go

```diff
@@ -63,11 +63,6 @@ func Configure(p *config.Provider) {
 		r.LateInitializer = config.LateInitializer{
 			IgnoredFields: []string{"threat_detection_policy"},
 		}
-		r.References = config.References{
```

Comment:
Thanks for removing these resource-group configurations. They must be belonging to pre-common reference configurations era.